Modules/Registration/Common/include/itkLandmarkBasedTransformInitializer.hxx

```diff
@@ -355,71 +361,180 @@ LandmarkBasedTransformInitializer<TTransform, TFixedImage, TMovingImage>::Intern
     itkExceptionMacro("Transform is VersorRigid3DTransform and Moving image dimension is not 3");
   }
 
-  // --- compute the necessary transform to match the two sets of landmarks
-  // ---
-  //
-  //
-  //    The solution is based on
-  //    Berthold K. P. Horn (1987),
-  //    "Closed-form solution of absolute orientation using unit
-  // quaternions,"
-  //    Journal of the Optical Society of America A, 4:629-642
-  //
-  //
-  //    Original python implementation by David G. Gobbi
-  //    Readpted from the code in VTK: Hybrid/vtkLandmarkTransform
-  //
-  //----------------------------------------------------------------------------
+  /*----------------------------------------------------------------------------
+     Compute the necessary transform to match the two sets of landmarks
```

Comment:
For comment blocks, the ITK SW guide recommends using // (p.374), so not sure why this was changed.